src/frontend/src/common/components/library/data_subview/components/CreateNSTreeModal/components/RadioLabel/strings.ts

```diff
@@ -0,0 +1,41 @@
+import { Pathogen } from "src/common/redux/types";
+import { PathogenStringsType } from "src/common/types/strings";
+
+interface RadioLabelStrings {
+  nonContextualizedBestFor: string;
+  nonContextualizedDescription: string;
+  nonContextualizedGoodFor: string;
+  nonContextualizedNotRecommended: string;
+  overviewBestFor: string;
+  overviewDescription: string;
+  overviewGoodFor1: string;
+  overviewGoodFor2: string;
```

Comment:
Overall I'm happy with this approach, but I wonder how we'll handle things like if a certain pathogen has 3 bullet points or just 1 bullet point. If the various options start to have significantly divergent meanings for different pathogens, I don't know how standardized the interface will really be. I'm not trying to offer a solution or actually think this is important to address now, just something to consider in case it does come up in the hypothetical future.

Reply:
I agree with the concern.  

Hopefully by keeping the strings directly with the component it will help.  Maybe we could do something like use a subset of the pathogens for the type rather than all of them so it would only require the pathogens that have that feature.

Another option would be to use a nested object rather than just a string and then each pathogen could define what is in the object.  So every pathogen would be required to have an object, but we wouldn't enforce the exact shape. 
 ie:
```
overviewGoodFor: {
   one: "some thing",
   two: "some other thing",
}
```